lang/en.js

```diff
@@ -136,6 +136,12 @@ var language = {
     },
     'hr': {
       'title': 'Croatian'
+    },
+    'zh': {
+      'title': 'Chinese (simplified)'
+    },
+    'hk': {
+      'title': 'Chinese (traditional)'
```

Comment:
Should the lang codes here be updated as well?
This would also apply to es.js, and all the other language files.

Reply:
Yes, i think we would have to update en.js and round trip

Reply:
Done